eel/src/lib.rs

```diff
@@ -546,6 +547,61 @@ impl LightningNode {
         }
     }
 
+    pub fn lnurl_withdraw(&self, lnurlw: &str, amount_msat: u64) -> Result<String> {
+        let client = lnurl::Builder {
+            proxy: None,
+            timeout: Some(10_000),
+        }
+        .build_blocking()
+        .map_to_permanent_failure("Failed to build LNURL client")?;
+        let response = client
+            .make_request(lnurlw)
+            .map_to_runtime_error(RuntimeErrorCode::LNURLError, "Failed to query server")?;
+        let response = match response {
+            LnUrlResponse::LnUrlPayResponse(_) => Err(runtime_error(
+                RuntimeErrorCode::LNURLError,
+                "Expected Withdraw response, got Pay resposne",
+            )),
+            LnUrlResponse::LnUrlWithdrawResponse(response) => Ok(response),
+            LnUrlResponse::LnUrlChannelResponse(_) => Err(runtime_error(
+                RuntimeErrorCode::LNURLError,
+                "Expected Withdraw response, got Channel response",
+            )),
+        }?;
+
+        let min = response.min_withdrawable.unwrap_or(1);
+        let max = response.max_withdrawable;
+        if amount_msat < min {
+            return Err(runtime_error(
+                RuntimeErrorCode::LNURLError,
+                format!("Expected: {amount_msat} msat is below min withdrawable: {min} msat"),
+            ));
+        } else if amount_msat > max {
+            return Err(runtime_error(
+                RuntimeErrorCode::LNURLError,
+                format!("Expected: {amount_msat} msat is above max withdrawable: {max} msat"),
+            ));
+        }
+
+        let description = String::new();
+        // TODO: Put info about the offer.
+        let metadata = "".to_string();
```

Comment:
What's the difference between `"".to_string();` and `String::new();`? Can't `description` and `metadata` be initiated in the same way?